src/api/v1beta1/feature_flags.go

```diff
@@ -25,12 +25,13 @@ import (
 )
 
 const (
-	PublicAnnotationPrefix                            = "alpha.operator.dynatrace.com/"
-	annotationFeaturePrefix                           = PublicAnnotationPrefix + "feature-"
+	DeprecatedFeatureFlagPrefix                       = "alpha."
+	FeatureFlagAnnotationPrefix                       = "operator.dynatrace.com/"
+	annotationFeaturePrefix                           = FeatureFlagAnnotationPrefix + "feature-"
 	annotationFeatureDisableActiveGateUpdates         = annotationFeaturePrefix + "disable-activegate-updates"
 	annotationFeatureDisableHostsRequests             = annotationFeaturePrefix + "disable-hosts-requests"
 	annotationFeatureOneAgentMaxUnavailable           = annotationFeaturePrefix + "oneagent-max-unavailable"
-	annotationFeatureEnableWebhookReinvocationPolicy  = annotationFeaturePrefix + "enable-webhook-reinvocation-policy"
+	AnnotationFeatureEnableWebhookReinvocationPolicy  = annotationFeaturePrefix + "enable-webhook-reinvocation-policy"
```

Comment:
I don't like it exported among other similar but not exported consts.
Maybe, to make it consistent, all annotations should be exported to make usable in unit tests of other packages?

Reply:
Made everything public to be consistent, and in a perfect we should have unittests for all of them